### ISSUE TITLE
Video save option

### ISSUE THREAD

**Issue by davidbhr** [rgerum/saenopy#14]
Create Videos of 3D-Fields and projections for timelines. Also for the 3D rendering, as soon as we have this. In case of only 2 timepoints Gifs with the reference state could be an option.

**Comment by rgerum** [comment]
the export of 2 timepoint gifs is still missing I think.

**Comment by davidbhr** [comment]
Right, currently the 2D images/projections can be saved separately in the stack dialog. An idea would be to merge the 2D images/projections (as an optional checkbox) into the "Save Images" dialog. Then everything would be in the same dialog and a ".gif" extension could automatically save either the timeline or the live-relaxed image pair (with e.g. 2 fps).

**Comment by rgerum** [comment]
Currently the .avi or .mp3 export does not work on my computer.
